client/my-sites/plugins/plugins-list/index.tsx

```diff
@@ -532,14 +575,18 @@ export class PluginsList extends Component {
 		const { plugins } = this.props;
 		return plugins.reduce( ( sites, plugin ) => {
 			Object.keys( plugin.sites ).map( ( pluginSiteId ) => {
-				if ( ! sites.find( ( site ) => site.ID === pluginSiteId ) ) {
-					const pluginSite = this.props.allSites.find( ( s ) => s.ID === parseInt( pluginSiteId ) );
-					sites.push( pluginSite );
+				if ( ! sites.find( ( site ) => site && site.ID === Number( pluginSiteId ) ) ) {
```

Comment:
I'd prefer using `some` to `find` if we don't need to utilize the found value.

```suggestion
				if ( ! sites.some( ( site ) => site && site.ID === Number( pluginSiteId ) ) ) {
```